docs/data-sources/mlflow_model.md

```diff
@@ -0,0 +1,35 @@
+---
+subcategory: "MLflow"
+---
+# databricks_mlflow_model Data Source
+
+-> **Note** If you have a fully automated setup with workspaces created by [databricks_mws_workspaces](../resources/mws_workspaces.md) or [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace), please make sure to add [depends_on attribute](../index.md#data-resources-and-authentication-is-not-configured-errors) in order to prevent _default auth: cannot configure default credentials_ errors.
+
+Retrieves the settings of [databricks_mlflow_model](../resources/mlflow_model.md) by name.
```

Comment:
In other data sources, getting by ID is also typically supported. Should we add that in this one as well?

Reply:
I reviewed the documentation, it looks like [fetching a single model](https://docs.databricks.com/api/workspace/modelregistry/getmodel) does return the ID. However, [listing models](https://docs.databricks.com/api/workspace/modelregistry/listmodels) in order to search by ID does not seem to be present.